addons/binding/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/handler/EnOceanBridgeHandler.java

```diff
@@ -250,13 +254,13 @@ public void responseReceived(RDVersionResponse response) {
                     });
 
         } catch (IOException e) {
-            logger.debug("error during bridge init occured", e);
+            logger.error("error during bridge init occured", e);
```

Comment:
No error log level please, that is reserved for the framework. Also no logging in front of any update* method. Those are logged already.

Reply:
Oh, you are quick :-) Would you please explain why error level should be framework exclusive? How should I fix that? Log with warn level, or re-raise an unchecked exception?

Reply:
> explain ... error level should be framework exclusive?

See Coding guidelines.

> How should I fix that? Log with warn

yes exactly.

Reply:
Ok thanks, I have changed that.